### DISCUSSION TITLE
AbogaBoot

### DISCUSSION THREAD

**Discussion by LeonardoSoriano** [Launch-X-Latam/MisionFrontEnd#111] [General]
Por ahora solo es hacer los bocetos y el wireframe? o también llenamos de una vez el word sobre los requerimientos?

**Comment by samujm** [comment]
Tambien el documento, tiene otros requisitos dentro, dale una checada, tienes que realizar la plantilla de tu grupo de personas objetivo, bocetos wireframe y UI

**Reply by muisca2020** [reply]
Los creamos y tenemos que entregarlos o cargarlos en algún lado? Saben?

**Comment by Neo651** [comment]
Recien inicio , gracias
